src/WormholeJoin.sol

```diff
@@ -161,7 +170,7 @@ contract WormholeJoin {
                             );
 
         uint256 fee = vatLive ? FeesLike(fees[wormholeGUID.sourceDomain]).getFee(wormholeGUID, line_, debt_, pending, amtToTake) : 0;
-        require(fee <= maxFeePercentage * amtToTake / WAD, "WormholeJoin/max-fee-exceed");
+        if (fee > maxFeePercentage * amtToTake / WAD) revert MaxFeeExceeded(fee, maxFeePercentage * amtToTake / WAD);
```

Comment:
`maxFeePercentage * amtToTake / WAD`  should be cached to `maxFee`.

Reply:
done in 0172295